nixos/modules/security/acme.xml

```diff
@@ -66,4 +66,23 @@ options for the <literal>security.acme</literal> module.</para>
 
 </section>
 
+<section><title>Using ACME certificates in Nginx</title>
+<para>In practice ACME is mostly used for retrieval &amp; renewal of certificates that will be used in a webserver like Nginx. A configuration for Nginx that uses the certificates from ACME for <literal>foo.example.com</literal> will look similar to:</para>
```

Comment:
Thanks, these are fixed now. `&amp;` is still being used else where in `acme.xml` though. I'll leave those for a separate PR.